priv/static/specs/http_api_spec.yaml

```diff
@@ -52,6 +52,28 @@ paths:
       responses:
         '200':
           description: returns the metrics
+  /contexts:
+    delete:
+      summary: 'delete all contexts'
+      responses:
+        '200':
+          description: return count of current contexts after deleting
+          schema:
+            $ref: '#/definitions/DeleteContexts'
+  /contexts/{count}:
+    delete:
+      summary: 'delete N contexts'
+      parameters:
+        - name: count
+          in: path
+          required: true
+          type: integer
+          description: 'number of contexts for deleting'
+      responses:
+        '200':
+          description: return count of current contexts after deleting
```

Comment:
That is not correct. The operation is asynchronous, it only triggers a context shutdown. At the moment the count is take, not all shutdowns need to have completed. The number returned could therefore be higher than expected.